doc/Windows.md

```diff
@@ -26,3 +26,23 @@ mvn quarkus:dev -Dquarkus.quinoa=false
 
 ### 4. Access the development server
 * This is the port for the application [localhost:8080](http://localhost:8080)
+
+### Access the frontend development server
+
+To access the frontend development server, you need to:
+
+1. Install the dependencies by running the following command:
+
+```
+npm install
+```
+
+
```

Comment:
### Access the frontend development server
To access the frontend development server, you need to install the dependencies by running the following command

```
npm install
```

Reply:
Let me do this right now, thank you.

Reply:
This should be implemented like this